app/Repositories/PhotoRepository.php

```diff
@@ -154,17 +155,48 @@ public function reviews($data)
         foreach ($data as $review) {
             if (isset($review['rogue_event_id']) && ! empty($review['rogue_event_id'])) {
                 $post = Post::where(['event_id' => $review['rogue_event_id']])->first();
+                $photo = Photo::where(['id' => $post->postable_id])->first();
 
                 if ($review['status'] && ! empty($review['status'])) {
                     // @TODO: update to add more details in the event e.g. admin who reviewed, admin's northstar id, etc.
                     $review['submission_type'] = 'admin';
 
-                    Event::create($review);
-
-                    $post->content->status = $review['status'];
-                    $post->content->save();
-
-                    array_push($reviewedPhotos, $post->content);
+                    // Create the Event.
+                    $event = Event::create([
```

Comment:
@sbsmith86 I don't feel strongly either way! Maybe keep for now since there is a test to make sure an event is created and we'll want to keep this test even when we switch to creation logic into a model event?